src/3box.js

```diff
@@ -245,6 +248,12 @@ class Box extends BoxApi {
     if (!this.spaces[space]) {
       this.spaces[space] = new Space(space, this.replicator)
     }
+
+    if (this._ghostPinbot) {
+      Object.assign(opts, {
+        ghostPinbot: this._ghostPinbot
+      })
```

Comment:
I'd just do `opts.ghostPinbot = this._ghostPinbot` easier to read :)

Reply:
done 👍